### PR TITLE
add synthetic badge and all answers wrong checkbox to ranking task

### PR DESCRIPTION
**Pull request by notmd**
image

### CODE REVIEW

File: website/public/locales/en/message.json [hunk 1]
```diff
@@ -23,5 +23,7 @@
   "submit_labels": "Submit",
   "tree_stopped": "Tree stopped {{id}}",
   "view_user": "View user",
-  "your_recent_messages": "Your Recent Messages"
+  "your_recent_messages": "Your Recent Messages",
+  "synthetic": "Synthetic",
+  "synthetic_explain": "This message is generated by our AI"
```
Comment (AbdBarho) [inline website/public/locales/en/message.json:28]: I think when people read "our" they will start to think  and ask if the models are ready for public use, which is not the case.

maybe we can remove "our"? leave it vague? This message is AI generated.

File: website/public/locales/en/tasks.json [hunk 2]
```diff
@@ -88,5 +88,6 @@
   "submitted_as": "This will be submitted as {{submit_lang}}",
   "tab_write": "Write",
   "tab_preview": "Preview",
-  "writing_wrong_langauge_a_b": "You appear to be writing in {{detected_lang}} but this will be submitted as {{submit_lang}}."
+  "writing_wrong_langauge_a_b": "You appear to be writing in {{detected_lang}} but this will be submitted as {{submit_lang}}.",
+  "not_rankable": "All answers are wrong and not rankable"
```
Comment (AbdBarho) [inline website/public/locales/en/tasks.json:92]: right / wrong is subjective, maybe something like:

All answers are factually incorrect and cannot be ranked.

File: website/src/components/Sortable/SortableItem.tsx [hunk 3]
```diff
@@ -62,6 +67,22 @@ export const SortableItem = ({
         onPointerDown={listeners?.onPointerDown as PointerEventHandler<HTMLDivElement>}
         className="w-[67%] lg:w-[80%]  h-full  absolute ltr:left-0 rtl:right-0 top-0 touch-none"
       ></div>
+      <Flex
+        position="absolute"
+        gap="2"
+        top="-2.5"
+        style={{
+          insetInlineEnd: "1.25rem",
```
Comment (AbdBarho) [inline website/src/components/Sortable/SortableItem.tsx:75]: ❤

File: website/src/components/Tasks/EvaluateTask.tsx [hunk 4]
```diff
@@ -18,29 +20,40 @@ export const EvaluateTask = ({
...
+    }
+  }, [task, ranking, onReplyChanged, onValidityChanged, notRankable]);
 
-  const sortables = task.type === TaskType.rank_initial_prompts ? "prompts" : "replies";
+  const handleNotRankableChange = useCallback((e: ChangeEvent<HTMLInputElement>) => {
+    setNotRankable(e.target.checked);
+  }, []);
 
+  const { t } = useTranslation("tasks");
+  // @notmd: I haven't test `rank_initial_prompts` type yet
```
Comment (AbdBarho) [inline website/src/components/Tasks/EvaluateTask.tsx:54]: what do you mean by this?

**Review by AbdBarho — COMMENTED** [review]
Thanks, looks great

File: website/public/locales/en/tasks.json (hunk 2 above)
Comment (notmd) [reply website/public/locales/en/tasks.json]: Thanks for suggestion, I'm really bad at these stuffs, lol

File: website/src/components/Tasks/EvaluateTask.tsx (hunk 4 above)
Comment (notmd) [reply website/src/components/Tasks/EvaluateTask.tsx:54]: It means I haven't tested the `rank_initial_prompts` task yet. We currently don't enable that task in the backend.

File: website/public/locales/en/message.json (hunk 1 above)
Comment (notmd) [reply website/public/locales/en/message.json]: done.

File: website/src/components/Tasks/EvaluateTask.tsx (hunk 4 above)
Comment (andreaskoepf) [reply website/src/components/Tasks/EvaluateTask.tsx:54]: Currently there are no plans to re-enable initial prompt ranking.

**Review by AbdBarho — APPROVED** [review]

**Pull request merged**